File: Numbers/LeapYear.java
public class LeapYear {
    public static boolean checkLeapYear(int n){
        int Year=n;
        if((Year%400==0)||
            (Year%100!=0)&&
            (Year%4==0)){
                return true;
            }else{
                return false;
            }
    }

    public static void main(String[] args) {
        int yr=1993;
        if(checkLeapYear(yr)){
            System.out.println(yr+":is a Leap Year");
        }else{
            System.out.println(yr+":is not a Leap Year");
        }
    }
    
}

File: Problem Solving Python/Numbers/LeapYear.py
def LeapYear(n):
    if(n%400==0 or n%100!=0 and n%4==0):
        return True
    else:
        return False

if __name__=='__main__':
    n=int(input("Enter a year to check leap-year: "))
    if(LeapYear(n)):
        print(f"The year {n} is a leap-year")
    else:
         print(f"The year {n} is not a leap-year")
